.github/workflows/release.yaml

```
@@ -44,6 +45,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
   ship-docker-image:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Since the Workflow doesn’t run on `master` only anymore, we have to ensure this job doesn’t run in other branches / Pull Requests:

```suggestion
    if: github.ref == 'refs/heads/master' && github.event.workflow_run.conclusion == 'success'
```